### PR TITLE
Dismemberment fixes!!! (fixes A LOT of things, mostly #2236)

### PR DESCRIPTION
**Pull request by Crystalwarrior**
- bugfix: "Fixed double e-swords not counting as sharp and, in turn, be unable to dismember"
- bugfix: "Fixes skeletons having colors"
- bugfix: "Fixes augmentation being weird"
- bugfix: "Fixes examinetext for self-examinig damage having incorrect grammar"
- bugfix: "Fixed \"blood-stained hands\" flavortext oddities when missing arm(s)"
- bugfix: "Fixed tablesaws not doing proper dismemberment"
- bugfix: "Fixed being able to pick items up in severed-muscles-hand"
- bugfix: "Fixed limb icons during surgery"
- bugfix: "Fixed chest \"dismemberment\" (disemboweling) not giving you instant death when you lose vital organs (like heart)"
- bugfix: "Fixed newly-made limbs via limbspawn plants having missing icons"
- bugfix: "Fixed mutation colors via DNA/species"
- bugfix: "FIXED LIZARD TAILS AND OTHER MUTANT BODYPARTS (BLAME TG FOR THIS ONE. I'M SERIOUS.)"
- bugfix: "FIXED AUGMENTS BEING UNHEALABLE HOLY SHIT. BLAME ME FOR THIS ONE."
- tweak: "Gave defibs a seperate warning message if missing heart"
- soundadd: "Added skeleton species punch sound from pre-rebase!"

### CODE REVIEW

**Comment by Crystalwarrior** [comment]
https://github.com/HippieStationCode/HippieStation13/issues/2236

File: code/game/objects/items/weapons/defib.dm [hunk 1]
```diff
@@ -489,7 +489,9 @@
 
 						if (H.suiciding || (H.disabilities & NOCLONE))
 							failed = "<span class='warning'>[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Recovery of patient impossible. Further attempts futile.</span>"
-						else if ((tplus > tlimit) || !H.getorgan(/obj/item/organ/internal/heart))
+						else if (H.getorgan(/obj/item/organ/internal/heart))
```
Comment (JohnGinnane) [inline code/game/objects/items/weapons/defib.dm]: Are you saying if you find a heart then fail?

Comment (Crystalwarrior) [inline code/game/objects/items/weapons/defib.dm]: On defibs, yes. I only seperated the time check from heart missing for more info.

Comment (TDcoolguy300) [inline code/game/objects/items/weapons/defib.dm]: Yeah, whats up with that Crystal?

**Comment by Crystalwarrior** [comment]
Hold on, DNM, double eswords still don't dismember...

File: code/modules/surgery/limb_manipulation.dm [hunk 2]
```diff
@@ -123,6 +125,7 @@
 		if(istype(I))
 			I.state_flags = ORGAN_AUGMENTABLE
 			I.update_organ_icon()
+			target.regenerate_icons()
```
Comment (JohnGinnane) [inline code/modules/surgery/limb_manipulation.dm:128]: I think the three instances of "target.regenerate_icons()", as well as the lines above and below can be just put once after all of this, no? That way there's less repetition

**Comment by TDcoolguy300** [comment]
Other than the defib, it looks fine to me

**Comment by Crystalwarrior** [comment]
...Whats wrong with defib?

**Comment by JohnGinnane** [comment]
If there's a heart then it says "couldn't locate heart"

**Comment by TDcoolguy300** [comment]
^

**Comment by Crystalwarrior** [comment]
o o p s .

**Comment by Spacedong** [comment]
merge it cunt

**Comment by Crystalwarrior** [comment]
Can merge now once travis approves

**Comment by TDcoolguy300** [comment]
Not until we code check it for sure Spacecunt

**Comment by TDcoolguy300** [comment]
Codes fine now, i'll go and do a merge

**Comment by blobbernaut** [comment]
PogChamp

**Pull request merged**